docs/src/man/importing_and_exporting.md

```diff
@@ -113,6 +113,28 @@ As you can see the code required to transform `iris` into a proper input to the
 format is not easy. Therefore CSV.jl is the preferred package to write CSV files
 for data stored in data frames.
 
+## JLD2 Files
+
+A convenient format for reading and writing data frames in JLD2, which saves and loads
+Julia data structures in a format comprising a subset of HDF5. Unlike several other 
+formats, JLD2 preserves custom Types. The `save` and `load` functions, provided by FileIO.jl, 
+allow to read/write a data frame from/to a JLD2 file.  
```

Comment:
```suggestion
JLD2 is a HDF5-compatible file format that allows reading and writing data frames.
A valuable feature of JLD2 format is that it preserves custom column types of the stored data frame.

The `save` and `load` functions, provided by FileIO.jl, allow to read/write a data frame from/to a JLD2 file.  
```

Reply:
This looks good... just an FYI FileIO.jl *or* JLD2.jl can be used for read/write of JLD2 files.. but I think it's best to point the usere to a single package FileIO.jl

Reply:
updated